src/Compilers/CSharp/Test/Syntax/Parsing/PatternParsingTests.cs

```diff
@@ -7564,5 +7564,42 @@ public void BrokenSwitchExpression_03()
             }
             EOF();
         }
+
+        [Fact, WorkItem(33675, "https://github.com/dotnet/roslyn/issues/33675")]
+        public void ParenthesizedNamedConstantPatternInSwitchExpression()
+        {
+            UsingExpression("e switch { (X) => 1 }");
```

Comment:
We are considering adding onepls at some point, right? Is this going to interfere with being able to pattern match on them?

Reply:
No.  Parsing oneple patterns will require some other disambiguating syntax.